src/main/io/serial_cli.c

```diff
@@ -509,6 +511,59 @@ static bool isEmpty(const char *string)
     return *string == '\0';
 }
 
+static void cliRxFail(char *cmdline)
+{
+    enum { RXFAIL_ARGUMENT_COUNT = 2 };
+    int16_t arguments[RXFAIL_ARGUMENT_COUNT];
+    int i;
+    char *ptr;
+
+    if (isEmpty(cmdline)) {
+        // print out rxConfig failsafe settings
+        for (i = 0; i < MAX_SUPPORTED_RC_CHANNEL_COUNT; i++) {
+            printf("rxfail %u %d\r\n", i, masterConfig.rxConfig.rx_fail_usec[i]);
+        }
+    } else {
+        int validArgumentCount = 0;
+
+        ptr = cmdline;
+
+        // If command line doesn't fit the format, don't modify the config
+        while (*ptr) {
+            if (*ptr >= '0' && *ptr <= '9') {
```

Comment:
could use atoi() and not bother checking the characters since atoi does that internally. see other `cli*()` methods